### PR TITLE
add FoxeerF405v2

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: common/source/docs/common-foxeerf405v2.rst [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+
+- Available from many retailers including `Foxeer <https://www.foxeer.com/foxeer-f405-v2-fc-reaper-55a-esc-8s-stack-video-switcher-servo-barometer-g-578>`__
+
+
+Specifications
+==============
+
+-  **Processor**
+
+   -  STM32HF405 32-bit processor
```
Comment (andyp1per) [inline common/source/docs/common-foxeerf405v2.rst:24]: No H

File: common/source/docs/common-foxeerf405v2.rst [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+   -  SERIAL6 -> UART6 (User)
+
+Any UART may be re-tasked by changing its protocol parameter.
+
+RC Input
+========
+
+RC input is configured on the R2 (UART2_RX) pin. It supports all RC protocols except PPM and SBUS. See :ref:`common-rc-systems` for details for a specific RC system. :ref:`SERIAL2_PROTOCOL<SERIAL2_PROTOCOL>` is set to "23", by default, to enable this.
+
+- SBUS connects to the SBUS which inverts the input and applies it to to R2 input.
```
Comment (andyp1per) [inline common/source/docs/common-foxeerf405v2.rst:173]: SBUS pad?

File: common/source/docs/common-foxeerf405v2.rst [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+
+OSD Support
+===========
+
+The FoxeerF405v2  supports OSD using :ref:`OSD_TYPE<OSD_TYPE>` 1 (MAX7456 driver). The defaults are also setup to allow DJI Goggle OSD support on UART4. Both the internal analog OSD and the DisplayPort OSD can be used simultaneously by setting :ref:`OSD_TYPE2<OSD_TYPE2>` = 5
+
+PWM Output
+==========
+
+The FoxeerF405v2 supports up to 8 PWM outputs. Outputs are available via two JST-SH connectors. All 8 outputs support DShot and bi-directional DShot, as well as all PWM types.
```
Comment (andyp1per) [inline common/source/docs/common-foxeerf405v2.rst:191]: 9 PWM outputs

Comment (andyp1per) [reply common/source/docs/common-foxeerf405v2.rst:191]: None support bi-directional dshot

File: common/source/docs/common-foxeerf405v2.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-foxeerf405v2.rst]: fixed

File: common/source/docs/common-foxeerf405v2.rst (hunk 2 above)
Comment (Hwurzburg) [reply common/source/docs/common-foxeerf405v2.rst]: fixed

File: common/source/docs/common-foxeerf405v2.rst (hunk 3 above)
Comment (Hwurzburg) [reply common/source/docs/common-foxeerf405v2.rst]: fixed

**Pull request merged**